src/60_Samples_%26_Templates/Housing.html

```diff
@@ -18,21 +15,51 @@
   <title>Housing</title>
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <!--
```

Comment:
Can you add a section `setup` before the imports.

Reply:
done